### DISCUSSION TITLE
[Request] 是否可以考虑优化浏览器缓存占用

### DISCUSSION THREAD

**Discussion by leeyoshinari** [lobehub/lobehub#2380] [General Question]
### 🥰 需求描述

已经把浏览器所有聊天记录全部清理了，浏览器缓存占用还是比较多，是否可以考虑优化

![image](https://github.com/lobehub/lobe-chat/assets/25929786/a19eb7ae-8ab2-474f-be9e-eb6639c95319)

### 🧐 解决方案

减少非必要的数据储存

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这个是网页内容的缓存吧？你清理掉以后每次都要重新加载网页内容，感觉没必要吧

[a comment by an automated account is not shown]

**Comment by leeyoshinari** [comment]
lobe 要比其他类似功能的网址的加载速度要慢，怀疑就是缓存文件太多了，有太多没必要的文件被缓存了

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这个怀疑没啥根据，觉得哪里慢可以提出来。我们每次优化，大家的体感都非常明显说有变快。

[a comment by an automated account is not shown]

**Comment by jooler** [comment]
> lobe 要比其他类似功能的网址的加载速度要慢，怀疑就是缓存文件太多了，有太多没必要的文件被缓存了

有缓存才会快，缓存的目的就是为了快，不然为啥搞缓存？？

[a comment by an automated account is not shown]
